docs/attributes-registry/db.md

```diff
@@ -179,25 +180,15 @@ This group defines attributes for Azure Cosmos DB.
 | `direct`  | Direct connection.              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `gateway` | Gateway (HTTP) connections mode | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Didn't we decide this is the default value if no connection mode is indicated?

```suggestion
| `gateway` | Gateway (HTTP) connection. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Reply:
We would always have a connection mode.